src/main/java/org/reaktivity/specification/kafka/internal/KafkaFunctions.java

```diff
@@ -1623,6 +1702,19 @@ public KafkaMergedDataExMatcherBuilder partition(
                 return this;
             }
 
+            public KafkaMergedDataExMatcherBuilder partition(
```

Comment:
Please call this from the 2-parameter overloaded `partition` method, passing `DEFAULT_LATEST_OFFSET` for the `latestOffset` argument.